DJ_App/src/main/java/com/google/sps/data/Event.java

```diff
@@ -0,0 +1,19 @@
+package com.google.sps;
+
+/** An item on a todo list. */
+public final class Event {
+
+  private final long tokenID;
+  private final String djName;
+  private final String location;
+  private final TimeRange timeRange;
+  private final String eventDescription;
+
+  public Event (long ID, String djName, String location, TimeRange timeRange, String eventDescription) {
+    this.tokenID = ID;
+    this.djName = djName;
+    this.location = location;
+    this.timeRange = timeRange;
+    this.eventDescription = eventDescription;
+  }
```

Comment:
I added eventID, and eventName. I thought it might be useful. Feel free to let me know if you don't think it was necessary.